test/integration/verify/controls/aws_ec2_images.rb

```diff
@@ -0,0 +1,11 @@
+title 'Test AWS EC2 Images in bulk'
+
+control 'aws-ec2-images-1.0' do
+
+  impact 1.0
+  title 'Ensure AWS EC2 Image has current properties'
+
+  describe aws_ec2_images do
```

Comment:
As mentioned [here](https://github.com/inspec/inspec-aws/pull/245/files?file-filters%5B%5D=.md#r470708208), we should demonstrate how to filter returned resources using `where` and reference this control in the resource doc.

Reply:
Latest commit has updated tests and docs to show how to use filtering